### ISSUE TITLE
The terminal shows bugs after the app builds on ubuntu with `curl`

### ISSUE THREAD

**Issue by lamiazar** [socketsupply/socket#363]
@mribbons I was able to get `ssc` installed with the new `curl` command 
the app window opens shows with `ssc build -r -o`, but I see some bugs printed on the terminal
```
ok - done. type 'ssc -h' for help
mia@mia:~/socket/socket$ ssc -v
0.2.1 (bd6232e7)
mia@mia:~/socket/socket$ mkdir app
mia@mia:~/socket/socket$ cd app
mia@mia:~/socket/socket/app$ ssc init
• socket.ini created in /home/mia/socket/socket/app +6ms
• src/index.html created in /home/mia/socket/socket/app +0ms
• .gitignore created in /home/mia/socket/socket/app +0ms
mia@mia:~/socket/socket/app$ ssc build -r -o
• preparing build for linux +3ms
• WARNING: [linux] icon '/home/mia/socket/socket/app/src/icon.png' does not exist +11ms
• package prepared +0ms
• compiled native binary +3768ms
• Running App: /home/mia/socket/socket/app/build/linux/beepboop-dev_v1.0.0-1_amd64/opt/beepboop-dev/beepboop-dev --from-ssc +0ms

(beepboop-dev:21591): Gdk-CRITICAL **: 10:02:15.868: gdk_monitor_get_workarea: assertion 'GDK_IS_MONITOR (monitor)' failed

• Creating Window#0 +0ms
• Showing Window#0 (seq=) +677ms
*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug


*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug


*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug


*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug


*** BUG ***
In pixman_region32_init_rect: Invalid rectangle passed
Set a breakpoint on '_pixman_log_error' to debug



(beepboop-dev:21591): Gtk-WARNING **: 10:03:12.412: Negative content width -6 (allocation 18, extents 12x12) while allocating gadget (node label, owner GtkLabel)

^CSegmentation fault (core dumped)
```

**Comment by lamiazar** [comment]
![Screenshot from 2023-04-18 10-26-35](https://user-images.githubusercontent.com/79177582/232808996-371e2189-fc1e-45ec-9320-1624058e80c3.png)

**Comment by mribbons** [comment]
@lamiazar Does the app window show "Hello World?"

The output on Linux is quite noisy, they are just debug output messages from upstream dependencies.

It looks like you pressed CTRL+C? Does the app quit normally if you click `x`?

**Comment by lamiazar** [comment]
> @lamiazar Does the app window show "Hello World?"

yes it shows `Hello world`, but I need to maximize the app window to see it. When I run `ssc build -r -o` it opens like I showed in the screenshot above.

**Comment by lamiazar** [comment]
> Does the app quit normally if you click x?

yes it closes normally

**Comment by jwerle** [comment]
Fixed in #398
